src/authorization.ts

```diff
@@ -28,4 +33,8 @@ export interface Authorization {
      * Used to authorize Bundle transactions
      */
     isBundleRequestAuthorized(request: AuthorizationBundleRequest): Promise<boolean>;
+    /**
+     * Returns the resourceTypes for which the requester is allowed to perform the given operation.

```

Comment:
```suggestion
     * Returns the resourceTypes for which the requester is allowed to view the given operation.

```

Reply:
I may be missing something, but "view the operation" doesn't sound right to me. "perform the operation" seems better.